frontend/src/console/components/PreviewComponent/PreviewComponent.tsx

```diff
@@ -18,38 +19,42 @@ export const PreviewComponent: React.FC<PreviewComponentProps> = (
   const { instanceData } = props;
   const themeData = instanceData.draftThemeConfig!;
 
-  return (
-    <div className="theme-preview">
-      <Row className="theme-preview-navigation">
-        <Col className="theme-preview-navigation">
-          <NavigationMenu instanceData={instanceData} themeData={themeData} />
-        </Col>
-      </Row>
-      <div className="theme-home">
-        <Row className="theme-hero-container">
-          <Col>
-            <HeroPreview
-              heroCoverImage={instanceData.heroCoverImage || ''}
-              homePageHeroTitleColor={themeData.homePageHeroTitleColor}
-              homePageHeroSubtitleColor={themeData.homePageHeroSubtitleColor}
-              homepageOverlayHtml={
-                instanceData!.draftStaticContentOverrides &&
-                instanceData.draftStaticContentOverrides.homepageOverlayHtml
-              }
-            />
+  // To Do: Edit this to taste when implementing the other previews
+  if (props.currentPreview === 'dashboard') {
+    return (
+      <div className="theme-preview">
+        <Row className="theme-preview-navigation">
+          <Col className="theme-preview-navigation">
+            <NavigationMenu instanceData={instanceData} themeData={themeData} />
           </Col>
         </Row>
-        <Row className="theme-courses-container">
-          <Col md={3} className="theme-courses-item">
-            <CoursesListingItem themeData={themeData} />
+        <div className="theme-home">
+          <Row className="theme-hero-container">
+            <Col>
+              <HeroPreview
+                heroCoverImage={instanceData.heroCoverImage || ''}
+                homePageHeroTitleColor={themeData.homePageHeroTitleColor}
+                homePageHeroSubtitleColor={themeData.homePageHeroSubtitleColor}
+                homepageOverlayHtml={
+                  instanceData!.draftStaticContentOverrides &&
+                  instanceData.draftStaticContentOverrides.homepageOverlayHtml
+                }
+              />
+            </Col>
+          </Row>
+          <Row className="theme-courses-container">
+            <Col md={3} className="theme-courses-item">
+              <CoursesListingItem themeData={themeData} />
+            </Col>
+          </Row>
+        </div>
+        <Row className="theme-footer">
+          <Col>
+            <FooterPreview instanceData={instanceData} themeData={themeData} />
           </Col>
         </Row>
       </div>
-      <Row className="theme-footer">
-        <Col>
-          <FooterPreview instanceData={instanceData} themeData={themeData} />
-        </Col>
-      </Row>
-    </div>
-  );
+    );
+  }
+  return <div className="theme-preview" />;
```

Comment:
Can this element ever be returned? If not, this line can be dropped.

Reply:
es-lint expects all code paths to return a value